src/test/java/org/stellar/sdk/MemoTest.java

```diff
@@ -74,7 +74,41 @@ public void testParseMemoId() {
         MemoId memoId = (MemoId)transactionResponse.getMemo();
         assertEquals(longId, Long.toUnsignedString(memoId.getId()));
     }
+    
+    @Test
+    public void testParseMemoHash() {
```

Comment:
I don't think need this test case as it's going through code outside the target scope, `testMemoNullHexHashSuccess` exercises the same target code path correct?

Reply:
I've added this one since it's the initial case from the bug report and what can be found on the blockchain.
testMemoNullHexHashSuccess shows that it is possible to create such a memo.

Reply:
Ok, in that case, it's probably worth re-locate this unit test to that target, namely the `TransactionDeserializerTest`, looks like there are a few test cases on memo already, could drop this one there also:

https://github.com/vinamogit/java-stellar-sdk/blob/fix-empty-memohash-exception/src/test/java/org/stellar/sdk/responses/TransactionDeserializerTest.java#L93